allennlp/tests/common/testing.py

```diff
@@ -0,0 +1,19 @@
+import torch
+
+from allennlp.common.testing import AllenNlpTestCase, multi_device
+
+
+class TestFromParams(AllenNlpTestCase):
```

Comment:
Class name here needs updating.

(I came to see what Mark thought looked cute, noticed a copy-paste bug.)

Reply:
Thanks! Good catch.

Reply:
I put `TestTesting` after the module name, hope it's fine.